### PR TITLE
Advertising student GSoC blogs.

### PR DESCRIPTION
**Pull request by agheata**
We decided to make student blogs mandatory this year. This PR is advertising the fact and gives the link to a how-to blog.

### CODE REVIEW

File: _activities/gsoc.md [hunk 1]
```diff
@@ -47,6 +47,10 @@ For further information, feel free to contact the HSF GSoC admins or join our op
 
 [![Gitter](https://badges.gitter.im/HSF/HSF-GSoC.svg)](https://gitter.im/HSF/HSF-GSoC?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
 
+### Student pages
+
+Student blogs are mandatory for CERN-HSF starting with 2022 GSoC edition! If you are a CERN-HSF student, please read this [blog]({{ site.baseurl }}/gsoc/2022/blogs.html)
```
Comment (graeme-a-stewart) [inline _activities/gsoc.md:52]: Needs a full stop at the end.

**Review by graeme-a-stewart — CHANGES_REQUESTED** [review]
Hi Andri - thanks a lot! Basically fine, but one missing full stop and maybe trim the intro text a wee bit (suggestion made).

**Comment by agheata** [comment]
Thanks @graeme-a-stewart I tried to address all your comments.

**Review by graeme-a-stewart — APPROVED** [review]
LGTM!

**Pull request merged**